### PR TITLE
Show custom 409 error message from Core

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Show custom 409 error message from Core

On Presentation deletion, if there is a conflict
show the custom message returned from the Core

Catch and save first error in batch
Throw first error in batch
Catch batch error and process if available
If processing returns true log formatted message

[stage-2]

## Motivation and Context
Fix for #2207 

## How Has This Been Tested?
Tested changes locally. Updated unit tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: web/scripts/components/scrolling-list/services/svc-batch-operations.js [hunk 1]
```diff
@@ -46,8 +46,9 @@ angular.module('risevision.common.components.scrolling-list')
 
           var _executeOperation = function (item, args) {
             method(item, args)
-              .catch(function (e) {
+              .catch(function (err) {
                 operations.hasErrors = true;
+                operations.error = operations.error || err;
```
[the remark this answers is not in the document]
Comment (alex-deaconu) [reply web/scripts/components/scrolling-list/services/svc-batch-operations.js:51]: @ezequielc great catch! Will fix, thanks.

**Comment by alex-deaconu** [comment]
@ezequielc the reset issue should be fixed in the latest commit. Thanks.

**Review by ezequielc — APPROVED** [review]
Looks good 👍 Thanks for the fix

**Pull request merged**